src/components/Projects.tsx

```diff
@@ -1,7 +1,243 @@
 import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import {
+  space,
+  SpaceProps,
+  layout,
+  LayoutProps,
+  background,
+  BackgroundProps,
+  FlexboxProps,
+  flexbox,
+  PositionProps,
+  position,
+  ColorProps,
+  color,
+  TypographyProps,
+  typography,
+  border,
+  BorderProps,
+} from "styled-system";
+import stairs from "./assets/project-page/stairs.png";
+import shell from "./assets/project-page/shell.png";
+import eye from "./assets/project-page/eye.png";
+import statue from "./assets/project-page/statue.png";
+import dragon from "./assets/project-page/dragon.png";
+import knife from "./assets/project-page/knife.png";
+import mask from "./assets/project-page/mask.png";
+import spider from "./assets/project-page/spider.png";
+import magnify from "./assets/project-page/magnify.png";
+
+type BgProps = BackgroundProps &
+  ColorProps &
+  LayoutProps &
+  FlexboxProps &
+  TypographyProps &
+  LayoutProps &
+  SpaceProps;
+
+const Background = styled.div<BgProps>`
+  ${background};
+  ${color};
+  ${layout};
+  ${flexbox};
+  ${space};
+`;
+
+type LogoProps = SpaceProps &
+  LayoutProps &
+  FlexboxProps &
+  PositionProps &
+  ColorProps &
+  SpaceProps &
+  TypographyProps &
+  BorderProps &
+  ColorProps &
+  BorderProps;
+
+const HomeLink = styled.a<LogoProps>`
```

Comment:
`LogoProps` here is overkill for all the components

eg. HomeLink only uses `${space}` so it only needs `SpaceProps`

You could create a type called HomeLinkProps = SpaceProps, and then use HomeLinkProps

I would do the same for all the other components as well, only passing them the props they actually use

Does that make sense?

Reply:
Yeah, makes sense. I thought I was being DRY doing it that way but I think this is a better way to catch unused props.

Reply:
And.. another thing that I thought might be useful is to start ordering the list of props in the same order as their typescript counterpart as so: 

<img width="360" alt="Screenshot 2020-07-08 at 16 35 57" src="https://user-images.githubusercontent.com/53219789/86932103-341f8900-c139-11ea-8da0-0072b8cf0089.png">

Easier for QA and helps me to pick up duplicates that shouldn't be in there quicker, spotted a few in there already.

Reply:
oh FAIL haha just spotted on in that screenshot

Reply:
I've just pushed again (sorry)